utils/crates-io/publish.rs

```diff
@@ -0,0 +1,171 @@
+//! mini-program for publishing packages to crates.io.
+use anyhow::Result;
+use cargo_metadata::MetadataCommand;
+use cargo_toml::{Dependency, Manifest, Value};
+use crates_io::Registry;
+use curl::easy::Easy;
+use std::{
+    collections::{BTreeMap, HashMap},
+    env, fs,
+    path::PathBuf,
+    process::{Command, ExitStatus},
+    thread,
+    time::Duration,
+};
+
+/// Packages need to be published.
+const PACKAGES: [&str; 14] = [
+    // Packages without local dependencies.
+    "actor-system-error",
+    "gear-common-codegen",
+    "gear-core-errors",
+    "gear-wasm-instrument",
+    "gmeta-codegen",
+    "gsdk-codegen",
+    "gsys",
+    // The packages below have local dependencies,
+    // and should be published in order.
```

Comment:
I thought that the main reason of publishing anything for us is to publish basic ecosystem libs: gsys, gcore, gstd, gtest, gclient, gear-wasm-builder, isnt it?